app/controllers/admin_users_account_suspensions_controller.rb

```diff
@@ -5,23 +5,23 @@
 # Copyright (c) 2018 UK Citizens Online Democracy. All rights reserved.
 # Email: hello@mysociety.org; WWW: http://www.mysociety.org/
 class AdminUsersAccountSuspensionsController < AdminController
-  before_filter :set_banned_user
+  before_filter :set_user
   before_filter :set_suspension_reason
 
   def create
-    if @banned_user.update(ban_text: @suspension_reason)
+    if @user.update(ban_text: @suspension_reason)
       flash[:notice] = 'The user was suspended.'
     else
       flash[:error] = 'Something went wrong. The user could not be suspended.'
     end
 
-    redirect_to admin_user_path(@banned_user)
+    redirect_to admin_user_path(@user)
   end
 
   private
 
-  def set_banned_user
-    @banned_user = User.find(params[:user_id])
+  def set_user
+    @user = User.find(params[:user_id])
```

Comment:
`@user` is reserved for the currently logged in user, which is a bit of an Alaveteli gotcha.

I suggest renaming to `@suspended_user`; easiest to do this with a few edits when rebasing instead of fixups, given later changes.